tests/cpp_unit_tests/test_current_sensor.cpp

```diff
@@ -164,7 +216,7 @@ TEST_CASE("Test current sensor") {
         CurrentSensor<symmetric_t> const current_sensor{{.id = 1,
                                                          .measured_object = 1,
                                                          .measured_terminal_type = MeasuredTerminalType::branch3_1,
-                                                         .angle_measurement_type = AngleMeasurementType::local_angle,
+                                                         .angle_measurement_type = AngleMeasurementType::global_angle,
```

Comment:
any particular reason why you don't do the loop over both angle measurement types here?